libs/coin-modules/coin-hedera/src/api/index.integ.test.ts

```diff
@@ -317,5 +319,45 @@ describe("createApi", () => {
         associatedTokenId: expect.any(String),
       });
     });
+
+    it.each(["desc", "asc"] as const)(
+      "returns paginated operations for account with high activity (%s)",
+      async order => {
+        const commonPagination = {
+          minHeight: 0,
+          limit: 10,
+          order,
+        } satisfies Pagination;
+
+        const [page1, pagingToken1] = await api.listOperations(
+          MAINNET_TEST_ACCOUNTS.withTokens.accountId,
+          commonPagination,
+        );
+
+        const [page2, pagingToken2] = await api.listOperations(
+          MAINNET_TEST_ACCOUNTS.withTokens.accountId,
+          {
+            ...commonPagination,
+            lastPagingToken: pagingToken1,
+          },
+        );
+
+        const firstPage1Timestamp = page1[0]?.tx?.date;
+        const firstPage2Timestamp = page2[0]?.tx?.date;
+        const page1Hashes = new Set(page1.map(op => op.tx.hash));
+        const page2Hashes = new Set(page2.map(op => op.tx.hash));
+        const hasOverlap = [...page2Hashes].some(hash => page1Hashes.has(hash));
+
+        // NOTE: this won't be equal to limit, because single Hedera transaction can generate multiple operations
+        expect(page1.length).toBeGreaterThanOrEqual(commonPagination.limit);
+        expect(page2.length).toBeGreaterThanOrEqual(commonPagination.limit);
+        expect(pagingToken1).not.toBeNull();
+        expect(pagingToken2).not.toBeNull();
+        expect(hasOverlap).toBe(false);
+        expect(firstPage1Timestamp).toBeInstanceOf(Date);
+        expect(firstPage2Timestamp).toBeInstanceOf(Date);
+        expect(firstPage1Timestamp > firstPage2Timestamp).toBe(order === "desc");
```

Comment:
We could be more severe checking that 

```suggestion
expect(lastPage1Timestamp > firstPage2Timestamp).toBe(order === "desc");
expect(firstPage1Timestamp < lastPage2Timestamp).toBe(order === "asc);
```